colorbox.install

```diff
@@ -4,6 +4,16 @@
  * Install, update and uninstall functions for the colorbox module.
  */
 
+/**
+ * Implements hook_enable().
+ */
+function colorbox_enable() {
+  backdrop_set_message(t('Thanks for installing Colorbox'));
```

Comment:
I don't think two messages are necessary here. Can we remove the thank you message, or add it as the first sentence of the other message?

Reply:
I see that you already did that. :+1: